src/js/core/MesosStream.js

```diff
@@ -1,6 +1,8 @@
 import { stream } from "@dcos/mesos-client";
+import { ReplaySubject } from "rxjs/ReplaySubject";
 
 export const MesosStreamType = Symbol("MesosStreamType");
 export default stream({ type: "SUBSCRIBE" }, "/mesos/api/v1?subscribe")
-  .publishReplay()
+  .repeat()
+  .multicast(() => new ReplaySubject())
```

Comment:
This is the same as publishReplay:

https://github.com/Reactive-Extensions/RxJS/blob/master/src/modular/observable/replay.js